### PR TITLE
Clean code

### PR DESCRIPTION
**Pull request by System25**
Code cleaned following the rules of Rubocop software.

### CODE REVIEW

File: libraries/SellUtil.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#
```
Comment (alfem) [inline libraries/SellUtil.rb]: "SellUtil.rb" looks like a  typo.

Comment (System25) [reply libraries/SellUtil.rb]: Fixed!

File: providers/jobids.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
 action :reset do
-  node.normal['gecos_ws_mgmt'][new_resource.recipe][new_resource.resource]['job_ids'] = []
-end
\ No newline at end of file
+  node.normal['gecos_ws_mgmt'][new_resource.recipe][new_resource.resource][
+    'job_ids'] = []
```
Comment (alfem) [inline providers/jobids.rb]: Strange line splitting

Comment (System25) [reply providers/jobids.rb]: Rubocop doesn't detect it as an offense to the Ruby style guide. I tried other ways of splitting this type of lines and Rubocop detected them as an offense.

Comment (alfem) [reply providers/jobids.rb]: ok

File: recipes/default.rb [hunk 3]
```diff
@@ -10,102 +10,94 @@
...
-end
+$locale = $locale.split('.')[0] if $locale.include? '.'
 Chef::Log.info("Locale is #{$locale}")
 
-
-# This should not be necessary, as wrapper is in new GECOS-Agent package. It is a transitional solution.
-Chef::Log.info("Installing wrapper")
-cookbook_file "gecos-chef-client-wrapper" do
-  path "/usr/bin/gecos-chef-client-wrapper"
+# This should not be necessary, as wrapper is in new GECOS-Agent package.
```
Comment (alfem) [inline recipes/default.rb]: It is a good moment to delete this duplicate wrapper creation.

Comment (System25) [reply recipes/default.rb]: Ok. Removed

**Pull request merged**